apps/full-site-editing/full-site-editing-plugin/mailerlite/subscriber-popup.php

```diff
@@ -0,0 +1,128 @@
+					wp_kses(
+						/* translators: %1$s - URL to manage the widget, %2$s - documentation URL. */
+						__( 'You need to enter your numeric account ID and UUID for the <a href="%1$s">Mailerlite Widget</a> to work correctly. <a href="%2$s" target="_blank">Full instructions</a>.', 'jetpack' ),
+						array(
+							'a' => array(
+								'href'  => array(),
+								'title' => array(),
+							),
+						)
+					),
+					esc_url( admin_url( 'widgets.php' ) ),
+					'https://support.wordpress.com/widgets/mailerlite'
+				) . '</p>';
+				echo esc_js( $args['after_widget'] );
+			}
+			return;
+		}
+
+		if ( wp_script_is( 'mailerlite-subscriber-popup', 'enqueued' ) ) {
+			return;
+		}
+
+		wp_register_script( 'mailerlite-universal', 'https://static.mailerlite.com/js/universal.js', array(), '20200521', true );
+		wp_enqueue_script(
+			'mailerlite-subscriber-popup',
+			plugins_url( 'subscriber-popup.js', __FILE__ ),
+			array( 'mailerlite-universal' ),
+			'20200521',
+			true
+		);
+		wp_localize_script(
+			'mailerlite-subscriber-popup',
+			'jetpackMailerliteSettings',
+			array(
+				'account' => esc_attr( $instance['account'] ),
+				'uuid'    => esc_attr( $instance['uuid'] ),
+			)
+		);
+
+	}
+
+	/**
+	 * Updates a particular instance of a widget.
+	 *
+	 * @param array $new_instance New settings for this instance as input by the user via WP_Widget::form().
+	 * @param array $old_instance   Old settings for this instance.
+	 *
+	 * @return mixed
+	 */
+	public function update( $new_instance, $old_instance ) { // @phpcs:ignore Generic.CodeAnalysis.UnusedFunctionParameter.FoundInExtendedClassAfterLastUsed
+		$instance['account'] = wp_kses( stripslashes( $new_instance['account'] ), array() );
+		$instance['uuid']    = wp_kses( stripslashes( $new_instance['uuid'] ), array() );
+
+		return $instance;
+	}
+
+	/**
+	 * Editable form in WP-admin.
+	 *
+	 * @param array $instance - settings.
+	 */
+	public function form( $instance ) {
+		$instance = wp_parse_args(
+			(array) $instance,
+			array(
+				'account' => '',
+				'uuid'    => '',
+			)
+		);
+
+		echo '
+		<p><label for="' . esc_attr( $this->get_field_id( 'account' ) ) . '">';
+		/* translators: link to documentation */
+		printf( wp_kses_post( __( 'Account ID <a href="%s" target="_blank">(instructions)</a>:', 'jetpack' ) ), 'https://en.support.wordpress.com/widgets/mailerlite' );
+		echo '<input class="widefat" id="' . esc_attr( $this->get_field_id( 'account' ) ) . '" name="' . esc_attr( $this->get_field_name( 'account' ) ) . '" type="text" value="' . esc_attr( $instance['account'] ) . '" />
+		</label></p>
+		<p><label for="' . esc_attr( $this->get_field_id( 'shelf' ) ) . '">' . esc_html__( 'UUID:', 'jetpack' );
+		echo '<input class="widefat" id="' . esc_attr( $this->get_field_id( 'uuid' ) ) . '" name="' . esc_attr( $this->get_field_name( 'uuid' ) ) . '" type="text" value="' . esc_attr( $instance['uuid'] ) . '" />';
+		echo '</label></p>
+		';
+	}
+}
+add_action(
```

Comment:
Can we use a named function here? When we use anonymous functions for hooks, it means other plugins/themes can't use `remove_action` if necessary.

Reply:
Done!